catalog/resourcemanagement/resourcemanagement-querymonitor-ui/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="
+       http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <bean id="queryPlugin" class="org.codice.ddf.resourcemanagement.query.QueriesPlugin" />
+
+    <bean id="queryMonitor" class="org.codice.ddf.resourcemanagement.query.service.QueryMonitor">
+        <argument ref="queryMonitorPlugin" />
+    </bean>
+
+    <reference id="queryMonitorPlugin" interface="org.codice.ddf.resourcemanagement.query.plugin.QueryMonitorPlugin" />
+
+    <service interface="org.codice.ddf.admin.application.plugin.ApplicationPlugin"
+             ref="queryPlugin"/>
+</blueprint>
```

Comment:
It might make the blueprint more readable if the items are listed in the order of dependence.